src/main/java/net/minecraftforge/fmp/client/multipart/IFastMSRPart.java

```diff
@@ -0,0 +1,17 @@
+package net.minecraftforge.fmp.client.multipart;
+
+import net.minecraftforge.fmp.multipart.IMultipart;
+
+/**
+ * Interface that adds {@link FastMSR} support to {@link IMultipart}.<br/>
+ */
+public interface IFastMSRPart extends IMultipart
+{
+    
+    /**
+     * Checks whether or not this part currently needs a {@link FastMSR} or it should just use a
+     * {@link MultipartSpecialRenderer}.
+     */
+    public boolean hasFastRenderer();
```

Comment:
Good idea - this was probably done as the interface was introduced mid-1.9 and might've broken mods?